tabular/src/autogluon/tabular/models/text_prediction/text_prediction_v1_model.py

```diff
@@ -1,19 +1,16 @@
 """Text Prediction Model based on Pretrained Language Model. Version 1"""
```

Comment:
Why not clarify in docstring this is simply a wrapper around AutoGluon's TextPredictor?